### DISCUSSION TITLE
grpcio

### DISCUSSION THREAD

**Discussion by FeodorFitsner** [flet-dev/flet#3157] [Packages] [answered]
HTTP/2-based RPC framework. Required by `firebase-admin` package.

**Comment by FeodorFitsner** [comment, marked as answer]
[`grpcio`](https://pypi.flet.dev/grpcio) is available in Flet 0.25.0 pre-release and up for both iOS and Android. Give it a try and let us know how that works.

**Reply by LegendaryPistachio** [reply]
Are there any tutorials? What is the tutorial website?

**Reply by LegendaryPistachio** [reply]
Links for grpcio
[grpcio-1.67.1-1-cp312-cp312-android_24_arm64_v8a.whl](https://pypi.flet.dev/-/ver_k1omu/grpcio-1.67.1-1-cp312-cp312-android_24_arm64_v8a.whl#sha256=07392f3fab229ac8b0e4035692eccc1c7d30900aa4ae269ed86cbf4e3ba5aa0e)
[grpcio-1.67.1-1-cp312-cp312-android_24_armeabi_v7a.whl](https://pypi.flet.dev/-/ver_h73zn/grpcio-1.67.1-1-cp312-cp312-android_24_armeabi_v7a.whl#sha256=73d0e65960c81123d6f5e4b814176373b947874f2605eb71ef10e53644562307)
[grpcio-1.67.1-1-cp312-cp312-android_24_x86.whl](https://pypi.flet.dev/-/ver_MgR9u/grpcio-1.67.1-1-cp312-cp312-android_24_x86.whl#sha256=fef41bce25f450c749063baabeccbf47ac565603e03991b35b7aa52ae298341c)
[grpcio-1.67.1-1-cp312-cp312-android_24_x86_64.whl](https://pypi.flet.dev/-/ver_1kfhSj/grpcio-1.67.1-1-cp312-cp312-android_24_x86_64.whl#sha256=43afdf8dfb46082a5b35faf5ca5ccd3879d738fa09a32672b4e4010e38e899bd)
[grpcio-1.67.1-1-cp312-cp312-ios_13_0_arm64_iphoneos.whl](https://pypi.flet.dev/-/ver_xF60a/grpcio-1.67.1-1-cp312-cp312-ios_13_0_arm64_iphoneos.whl#sha256=6d7dc275d5e3a274eff991db794c7df52e2a23cd323596972642e044be8025f9)
[grpcio-1.67.1-1-cp312-cp312-ios_13_0_arm64_iphonesimulator.whl](https://pypi.flet.dev/-/ver_156vMV/grpcio-1.67.1-1-cp312-cp312-ios_13_0_arm64_iphonesimulator.whl#sha256=07c2637f2c19abe4cd51959d1e00f77cf1e084557b83d0530456fb200cfd4882)
[grpcio-1.67.1-1-cp312-cp312-ios_13_0_x86_64_iphonesimulator.whl](https://pypi.flet.dev/-/ver_1Agjn2/grpcio-1.67.1-1-cp312-cp312-ios_13_0_x86_64_iphonesimulator.whl#sha256=f785bc5f84b35ebd6d0b8e28807edfba42b09dc79618a19844739843e079e85b)
